index.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+
+?>
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="normalize.css" type="text/css" />
+    <title>Joseph:Home</title>
+    <link rel="stylesheet" href="styles.css" type="text/css" />
+</head>
+<body>
+<nav>
+    <div class="navContainer">
+        <div class="logoName">
+            JaHolt
+        </div>
+        <ul>
+            <a href="contactPage/contactPage.php"><li>CONTACT</li></a>
```

Comment:
not that it should be part of this review, but technically this is invalid HTML. the `<a>` should go inside the `<li>`